build-system/tasks/runtime-test.js

```diff
@@ -155,6 +176,9 @@ function printArgvMessages() {
     firefox: 'Running tests on Firefox.',
     ie: 'Running tests on IE.',
     edge: 'Running tests on Edge.',
+    'chrome_canary': 'Running tests on Chrome Canary.',
+    'chrome_flags': 'Running tests on version of Chrome with the flags ' +
```

Comment:
This is the message that gets printed out when a certain flag has been detected on the command line. Instead of adding a static message to `argvMessages` for `chrome_flags`, you could dynamically print the list of flags that were actually parsed.

You could insert this after line 202 here:

https://github.com/ampproject/amphtml/blob/9eab7bf52a1648c3ab70f40d55aa3b7f15b8a59f/build-system/tasks/runtime-test.js#L196-L208

Something like...
```
    if (argv.chrome_flags) {
      log(green('Launching'), cyan(chromeBase), green('with flags'), cyan(formattedFlagList));
    }
```